src/main/java/org/folio/dew/batch/bursarfeesfines/service/BursarFilterEvaluator.java

```diff
@@ -40,7 +41,24 @@ public static boolean evaluate(
       );
     } else if (filter instanceof BursarExportFilterAmount) {
       BursarExportFilterAmount filterAmount = (BursarExportFilterAmount) filter;
-      return true;
+      int centFeeValue = account
+        .getAccount()
+        .getAmount()
+        .multiply(new BigDecimal("100"))
+        .intValue();
+
+      switch (filterAmount.getCondition()) {
+        case LESS_THAN:
+          return centFeeValue < filterAmount.getAmount();
+        case GREATER_THAN:
+          return centFeeValue > filterAmount.getAmount();
+        case LESS_THAN_EQUAL:
+          return centFeeValue <= filterAmount.getAmount();
+        case GREATER_THAN_EQUAL_:
```

Comment:
hmm, weird that there's an extra `_` appended to this...will look into this and see where this came from